### PR TITLE
fix: Annotate mythic aspects [skip ci]

### PR DESCRIPTION
**Pull request by ShadowCat117**
image

### CODE REVIEW

File: common/src/main/java/com/wynntils/models/items/annotators/game/AspectAnnotator.java [hunk 1]
```diff
@@ -20,14 +20,19 @@
 
 public class AspectAnnotator implements GameItemAnnotator {
     private static final Pattern ASPECT_PATTERN = Pattern.compile("^§(.)Aspect of .*$");
+    private static final Pattern MYTHIC_ASPECT_PATTERN = Pattern.compile("^§(.)(.*) Embodiment of .*$");
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/models/items/annotators/game/AspectAnnotator.java]:
```suggestion
    private static final Pattern ASPECT_PATTERN = Pattern.compile("^§(.)(?:Aspect|Embodiment) of .*$");
```

What do you think? Or is there another reason?

Comment (ShadowCat117) [reply common/src/main/java/com/wynntils/models/items/annotators/game/AspectAnnotator.java]: Changing to `§(.*)` works. Thanks

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/models/items/annotators/game/AspectAnnotator.java]: Wait a bit. Are you expecting multiple color codes there? Like mythic color + bold? Perhaps `(§.)+` is wiser then.

Comment (ShadowCat117) [reply common/src/main/java/com/wynntils/models/items/annotators/game/AspectAnnotator.java]: The mythic ones have an archetype name before the "Embodiment" like "Paladin's Embodiment of..."

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/models/items/annotators/game/AspectAnnotator.java]: Ah. `§. (?:(?:(.+) Embodiment)|(?:Aspect)) of ...` perhaps?

Comment (ShadowCat117) [reply common/src/main/java/com/wynntils/models/items/annotators/game/AspectAnnotator.java]: `^§(.)(?:(?:(.+) Embodiment)|(?:Aspect)) of .*$` works?

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/models/items/annotators/game/AspectAnnotator.java]: If it works, it works :)

**Review by kristofbolyai — APPROVED** [review]
Thanks!

**Pull request merged**